src/components/molecules/Loading/Loading.tsx

```diff
@@ -0,0 +1,18 @@
+import React, { FC } from "react";
+
+import LoadingIcon from "assets/icons/icon-loading.svg";
+
+import "./Loading.scss";
+
+export interface LoadingProps {
+  message?: string;
+}
+
+export const Loading: FC<LoadingProps> = ({ message }) => {
```

Comment:
`React.FC` instead of just `FC`.

Also, message should have a default value here. Probably `"Loading..."`

Reply:
used `React.FC`: https://github.com/sparkletown/sparkle/pull/1426/commits/880f966fdbeb06065f5946456902e13410b88582

and about default message. hmm. I wouldn't specify that. According to our styleguide there can be loading without text. 
http://sapiens.chat/sparkle/styleguide/

So, in case the text is not needed we can use this component.

what would you say?

Reply:
> and about default message. hmm. I wouldn't specify that. According to our styleguide there can be loading without text.

I guess it depends what is likely to be our more common pattern. If we are going to write "Loading..." all over the place then it probably makes sense to default the text and then potentially add another optional prop for `imageOnly` or similar.

If we're more likely to use it without text, then not setting the default might be a better idea.

cc // @mike-lvov Thoughts?

Reply:
@0xdevalias I would say that not having the default `loading` text makes sense just to handle a case, when there is no text at all. For other cases, I think we explicitly more specific message than `Loading...`